src/arithmetic/list_funcs/list_funcs.c

```diff
@@ -12,6 +12,9 @@
 #include"../../query_ctx.h"
 #include "../../datatypes/array.h"
 #include "../../util/rax_extensions.h"
+#include "../boolean_funcs/boolean_funcs.h"
+#include "../numeric_funcs/numeric_funcs.h"
+#include "../string_funcs/string_funcs.h"
```

Comment:
order include in length order

Reply:
Ok, I forgot that, thanks. Fixed in [d339863](https://github.com/RedisGraph/RedisGraph/pull/2605/commits/d339863f9bcf573c291ff9add6dcf877ab0f99be)